app/views/components/profile-card.pug

```diff
@@ -9,7 +9,7 @@
   .profile__user-info
     span.profile__handle
       - if (user.github !== undefined)
-        a(href=user.github.html_url, target='_blank') @#{user.github.login}
+        span #{user.github.login}
```

Comment:
I don't think you need another `span` here since it's already nested in a `span` with the class of `.profile__handle`.

Reply:
without added any tags and using it directly like this ` #{user.github.login}` does not renders the name in UI. For that I added a span tag

Reply:
@robertcoopercode updated my PR. Removed span tag